diff -up ldc-1.33.0-beta2-src/CMakeLists.txt.omv~ ldc-1.33.0-beta2-src/CMakeLists.txt
--- ldc-1.33.0-beta2-src/CMakeLists.txt.omv~	2023-06-13 23:44:05.764808672 +0200
+++ ldc-1.33.0-beta2-src/CMakeLists.txt	2023-06-13 23:44:58.424250851 +0200
@@ -693,7 +693,7 @@ build_d_executable(
     "${LDC_EXE_FULL}"
     "${LDC_D_SOURCE_FILES}"
     "${DFLAGS_BUILD_TYPE} ${DFLAGS_LDC}"
-    "${ALTERNATIVE_MALLOC_O};${LDC_LINKERFLAG_LIST}"
+    "${ALTERNATIVE_MALLOC_O};${LDC_LINKERFLAG_LIST};-lzstd"
     "${FE_RES}"
     "${LDC_LIB}"
     ${COMPILE_D_MODULES_SEPARATELY}
